config/jobs/kubernetes/kops/kops-config.yaml

```diff
@@ -160,7 +160,7 @@ presubmits:
         - --aws-cluster-domain=test-cncf-aws.k8s.io
         - --check-leaked-resources=false
         - --cluster=
-        - --extract=ci/k8s-stable2
+        - --extract=ci/k8s-stable1
```

Comment:
I'm guessing this is related to the `# manual-release-bump-required` comment above, but I don't understand why / what's going on here.  The job name references `stable2` and it's using the `release-1.12` branch, but this is using `stable1` which corresponds to `release-1.13`?

Reply:
@krzyzacy is fixing this.

Reply:
so - this is a job for kubernetes/kops repo - if you look at https://github.com/kubernetes/kops, they don't have up-to-date release branches, so I'm going to drop all the release branch checks. (the set up is incorrect)

Reply:
job removed